src/cityreader/cityreader.py

```diff
@@ -16,12 +28,11 @@
 # should not be loaded into a City object.
 cities = []
 
-def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
-    return cities
+def cityreader():
+  with open('.\src\cityreader\cities.csv')as f:
+  data = csv.reader(f)
+  for row in data:
+        print(row)
```

Comment:
good job reading the file however you would want to append the row in the cities list

so something like cities.append(row)